staging/src/kubevirt.io/client-go/api/v1/hyperconvergedconfig.go

```diff
@@ -0,0 +1,39 @@
+package v1
+
+// This code is copied from
+// https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/pkg/apis/hco/v1beta1/hyperconverged_types.go
+// in order to avoid dependency loops
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+// HyperConvergedConfig is the schema for pod configuration that is going to
+// be propagated by HCO to its managed operators.
+//
+// +k8s:openapi-gen=true
+type HyperConvergedConfig struct {
+	// nodeSelector is the node selector applied to the relevant kind of pods
+	// It specifies a map of key-value pairs: for the pod to be eligible to run on a node,
+	// the node must have each of the indicated key-value pairs as labels
+	// (it can have additional labels as well).
+	// See https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
+	// +kubebuilder:validation:Optional
+	// +optional
+	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
+
+	// affinity enables pod affinity/anti-affinity placement expanding the types of constraints
+	// that can be expressed with nodeSelector.
+	// affinity is going to be applied to the relevant kind of pods in parallel with nodeSelector
+	// See https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
+	// +kubebuilder:validation:Optional
+	// +optional
+	Affinity *corev1.Affinity `json:"affinity,omitempty"`
```

Comment:
@tiraboschi please note this is implemented as a pointer -- makes the code that needs to use this much cleaner. This differs from the HCO suggested defaults.

Reply:
I imagine it's fine to make changes as long as we haven't made a release of CDI

Reply:
Done: https://github.com/kubevirt/hyperconverged-cluster-operator/pull/785